front/src/modules/pipeline-processes/components/BoardActionBarButtonDeletePipelineProcesses.tsx

```diff
@@ -0,0 +1,47 @@
+import { getOperationName } from '@apollo/client/utilities';
+import { useRecoilState } from 'recoil';
+
+import { EntityTableActionBarButton } from '@/ui/components/table/action-bar/EntityTableActionBarButton';
+import { IconTrash } from '@/ui/icons/index';
+import { useDeleteManyPipelineProgressMutation } from '~/generated/graphql';
+
+import { GET_PIPELINES } from '../queries';
+import { boardItemsState } from '../states/boardItemsState';
+import { selectedBoardItemsState } from '../states/selectedBoardItemsState';
+
+export function BoardActionBarButtonDeletePipelineProcesses() {
+  const [selectedBoardItems, setSelectedBoardItems] = useRecoilState(
+    selectedBoardItemsState,
+  );
+  const [items, setItems] = useRecoilState(boardItemsState);
```

Comment:
Same here for convention, try to avoid using contextless naming like items or variable